main.go

```diff
@@ -102,6 +107,8 @@ func main() {
 		os.Exit(1)
 	}
 
+	flag.StringVar(&listenerMetricsAddr, "listener-metrics-addr", "8080", "The address applied to AutoscalingListener metrics server")
```

Comment:
`8080` or `:8080` the default value to `metrics-addr` is `:8080`.

Reply:
My mistake :relaxed: Thanks!